src/sass/style.scss

```diff
@@ -64,3 +64,31 @@ body {
 .grav-c-two-columns-text {
   margin: 2rem auto 0em auto;
 }
+
+// Temporary overrides - About page partner logos
+.grav-c-list-inline-row {
+  margin-bottom: $grav-sp-m;
+  justify-content: center;
+}
+
+.grav-c-list-inline-row>li {
+  flex: auto 0 0;
+  margin-right: $grav-sp-m;
```

Comment:
Please do not these margins here. The `margin-left` is redundant since the same value is already being set by Gravity's CSS. The `margin-right` causes an unbalanced look on small screens when the two logos stack into a single column, because it means there's a bigger margin to the right of the logos than there is on the left.

Removing those 2 lines of CSS still works - the only visual difference is a smaller gap between the two logos when they appear side-by-side on larger screens.

If enlarging that horizontal gap on larger screens was intentional, then you should do it as follows:
```sass
.grav-c-list-inline-row {
    // Negative margin on container to cancel out left margin of left-most logo(s)
    margin-left: -1 * $bigger-gap-size;
}

.grav-c-list-inline-row > li {
    margin-left: $bigger-gap-size;
}
```

Reply:
No probs I've added those now. Was this type of spacing in the style guide somewhere or going to be?